input/include/NovelRT/Input/KeyStateFrameChangeLog.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#ifndef NOVELRT_INPUT_KEYSTATEFRAMECHANGELOG_H
+#define NOVELRT_INPUT_KEYSTATEFRAMECHANGELOG_H
+
+#ifndef NOVELRT_INPUT_H
+#error NovelRT does not support including types explicitly by default. Please include Input.h instead for the Input namespace subset.
+#endif
```

Comment:
Are we updating the include structure for this in a separate submission? I don't remember.